web/templates/homepage/index.html.eex

```diff
@@ -59,18 +59,21 @@
   <% end %>
 </section>
 
-<section class="pa4 tc-ns pa1 ph3 pb4 pb5-l" id="suggestion_form">
-  <div class="br1 shadow-2 ph3 pt3 pb2 mt4 tl w-60-ns center">
+<section class="pa4 tc-ns tl-m pa1 ph3 ph5-m ph6-l pb4 pb5-l w-100" id="suggestion_form">
```

Comment:
do we want the width to be at 100% on desktop?
![screenshot from 2017-05-11 16-02-59](https://cloud.githubusercontent.com/assets/6057298/25956382/7bc48e86-3663-11e7-93f4-83804c7e9a48.png)